src/thunks/addNewNote.test.js

```diff
@@ -56,4 +57,11 @@ describe("addNewNote Thunk", () => {
     await thunk(mockDispatch);
     expect(mockDispatch).toHaveBeenCalledWith(actions.addNote(mockNotes[0]));
   });
+
+  it.skip("should return error when post fails", async () => {
```

Comment:
Is this working? It's skipped. Just checking if thats intentional.

Reply:
I think there is one test we decided to skip because it was having issues with the error and I'm pretty sure this was the one we skipped on purpose